docs/ContribOperators.md

```diff
@@ -4547,6 +4547,8 @@ This version of the operator has been available since version 1 of the 'com.micr
 <dl>
 <dt><tt>epsilon</tt> : float</dt>
 <dd>The epsilon value to use to avoid division by zero.</dd>
+<dt><tt>strict</tt> : int</dt>
```

Comment:
I think it is not good to change operator spec for this. Note that the op is used for other EP (like CPU etc). If every EP has some option added to operator interface, it will be hard to maintain.

I suggest using either environment variable (for internal testing) or provider option (for end user) for such purpose.